src/core/containers/info.jsx

```diff
@@ -19,14 +19,18 @@ export default class InfoContainer extends React.Component {
     const host = specSelectors.host()
     const externalDocs = specSelectors.externalDocs()
     const selectedServer = oas3Selectors.selectedServer()
+    const oasVersion = () => {
```

Comment:
You ideally don't want this here. In future we'll be separating non-plugin related code into plugins. This `info.jsx` component will be part of `oas2` plugin and it shouldn't know anything about other OpenAPI spec versions. Please have a look at https://github.com/swagger-api/swagger-ui/pull/9170/files how this is achieved.

Reply:
Right. I updated the implementation and separated non-plugin related code.